### ISSUE TITLE
Build a structured representation of the commemorative plaques in Jena

### ISSUE THREAD

**Issue by Daniel-Mietchen** [Daniel-Mietchen/ideas#557]
as per http://dbm.neuro.uni-jena.de/gedenktafel/

**Comment by Daniel-Mietchen** [comment]
Notes in https://etherpad.wikimedia.org/p/jenathon2017 .

**Comment by Daniel-Mietchen** [comment]
This is part of the Jenathon, a hackathon taking place in Jena today, as per https://github.com/Daniel-Mietchen/events/issues/166 .

**Comment by Daniel-Mietchen** [comment]
For this project, we're working in a team of three, with the following tasks:
- parsing http://dbm.neuro.uni-jena.de/gedenktafel/ for information on
  - the person's name
  - the corresponding Wikipedia article when available
  - residency address of that person in Jena
  - address of the commemorative plaques
- converting the address data from above into geolocation
  - using https://opendata.jena.de/dataset/adressen-verortet
- entering the information into Wikidata and visualizing it
  - converting Wikipedia articles to Wikidata items
    - via the Wikipedia API: https://en.wikipedia.org/w/api.php?action=query&prop=pageprops&ppprop=wikibase_item&redirects=1&titles=Franz_Liszt
  - setting up test items in Wikidata for two of the plaques to work out a data model for commemorative plaques in Jena
    - https://www.wikidata.org/wiki/Q42662495 (Michael Stifel)
    - https://www.wikidata.org/wiki/Q42661861 (Adam Drese)
  - setting up a Wikidata query to visualize the plaques and the corresponding people on a map
  - setting up a workflow to get the parsed information about the plaques into Wikidata

**Comment by Daniel-Mietchen** [comment]
Rough data model for commemorative plaques:
```
Q42662495	Len	"Michael Stifel commemorative plaque"
Q42662495	Den	"commemorative plaque in Jena"
Q42662495	Lde	"Michael-Stifel-Gedenktafel in Jena"
Q42662495	Dde	"Gedenktafel in Jena"
Q42662495	P31	Q721747
Q42662495	P17	Q183
Q42662495	P131	Q3150
Q42662495	P625	@LAT/LON
Q42662495	P547	Q41309
Q42662495	P969	"Kollegiengasse 10, 07743 Jena"
```

**Comment by Daniel-Mietchen** [comment]
There is also a dedicated category on Wikimedia Commons for commemorative plaques in Jena:
https://commons.wikimedia.org/wiki/Category:Plaques_in_Jena .

**Comment by Daniel-Mietchen** [comment]
Here's some data extracted from the dataset, and I'll try now to upload it via QuickStatements.
```
CREATE
LAST	P31	Q721747
LAST	P17	Q183
LAST	P131	Q3150
LAST	Len	"Richard Hertwig commemorative plaque"
LAST	Den	"commemorative plaque in Jena"
LAST	Lde	"Richard-Hertwig-Gedenktafel in Jena"
LAST	Dde	"Gedenktafel in Jena"
LAST	P625	@50.9325513300518/11.5835609324724
LAST	P547	Q85907
LAST	P969	"Philosophenweg 3"
```

**Comment by Daniel-Mietchen** [comment]
This worked: https://www.wikidata.org/wiki/Q42674252 .

**Comment by Daniel-Mietchen** [comment]
The [SPARQL query](http://tinyurl.com/y764escw) to get the commemorative plaques in Jena from Wikidata:

```SPARQL
#defaultView:Map
SELECT ?coord ?person ?personLabel ?imageT ?imageP ?tafel ?tafelLabel WHERE {
  ?tafel wdt:P31 wd:Q721747;
         wdt:P625 ?coord;
         wdt:P131 wd:Q3150 ;
         wdt:P547 ?person .
  OPTIONAL { ?tafel wdt:P18 ?imageT }.
  OPTIONAL { ?person wdt:P18 ?imageP }.
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],de,en". }
}
```

**Comment by SirkoS** [comment]
Added parsing/matching/... scripts to repo: https://github.com/SirkoS/jenathon2-plaques
Current results: https://gist.github.com/SirkoS/d7e8c778cb34a77878ab7f7ccaa776db

current parsing result of source page still contains some errors:
* incorrect column counts in some rows
* invalid addresses: some entries (in the source page) just have a `W` entries, which makes the parser fail

**Comment by Daniel-Mietchen** [comment]
OK, testing with
```
CREATE
LAST	P17	Q183
LAST	P131	Q3150
LAST	Len	"commemorative plaque: Abbe, Ernst"
LAST	Lde	"Gedenktafel in Jena: Abbe, Ernst"
LAST	Den	"commemorative plaque in Jena"
LAST	Dde	"Gedenktafel in Jena"
LAST	P547	Q76359
```

**Comment by Daniel-Mietchen** [comment]
Here we go: https://www.wikidata.org/wiki/Q42718673 .
The res_wikidataBatch.txt had two spaces instead of tabs - fixed that and uploading now.

**Comment by Daniel-Mietchen** [comment]
There was the statement missing that these items are instances of commemorative plaques:
```
Q42718673	P31	Q721747
```
Fixed now.

**Comment by SirkoS** [comment]
Fixed that missing property in the generating script.
The spaces instead of tabs issue does not appear for me. Both local and gist file seem fine.

I'll create a new output file, when we have a better parse result from the original or do you want a new one based on the current data?

**Comment by Daniel-Mietchen** [comment]
Yes, once we have better parser results, updates to the output file would be useful.
For plaques already in Wikidata, the edits should not use CREATE & LAST any more, but their respective Wikidata ID. The full list of plaques in Jena is returned by [this query](http://tinyurl.com/y9kdrpkl).

**Comment by Daniel-Mietchen** [comment]
Here is a [query](http://tinyurl.com/y8v39tpu) for getting images of the people commemorated by the plaques:

```SPARQL
SELECT ?coord ?person ?personLabel ?imageT ?imageP ?tafel ?tafelLabel WHERE {
  ?tafel wdt:P31 wd:Q721747;  # instances of commemorative plaques
         wdt:P131 wd:Q3150 ;  # located in the administrative territorial entity of the city of Jena
         wdt:P547 ?person .   # commemorating a person
#  OPTIONAL { ?tafel wdt:P18 ?imageT }.  # image of the plaque
#  OPTIONAL { ?tafel wdt:P625 ?coord }.  # coordinate locations
  OPTIONAL { ?person wdt:P18 ?imageP }. # image of the person
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],de,en". } # labels for the plaque and the person
}
#defaultView:ImageGrid
```

**Comment by Daniel-Mietchen** [comment]
Just stumbled across some tweets from the event:
- https://twitter.com/EvoMRI/status/927352639381176320
- https://twitter.com/EvoMRI/status/927201764302049281
